Add CQCM staging server to platforms to share data with

mkllnk opened the pull request:
#### What? Why?

You should now be able to share your enterprise with https://cqcm-map.startinblox.com/.

#### What should we test?
<!-- List which features should be tested and how.
     This can be similar to the Steps to Reproduce in the issue.
     Also think of other parts of the app which could be affected
     by your change. -->

- Enable feature `cqcm-stg`.
- Go to edit an enterprise.
- Click on the Connected Apps tab.
- Share your data with the staging platform.
- ??? Any other manual approvals???

#### Release notes

<!-- Please select one for your PR and delete the other. -->

Changelog Category (reviewers may add a label for the release notes):

- [ ] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [ ] Technical changes only
- [x] Feature toggled

<!-- Choose a pull request title above which explains your change to a
     a user of the Open Food Network app. -->

The title of the pull request will be included in the release notes.


#### Dependencies
<!-- Does this PR depend on another one?
     Add the link or remove this section. -->



#### Documentation updates
<!-- Are there any wiki pages that need updating after merging this PR?
     List them here or remove this section. -->

filipefurtad0 commented:
Ok, thanks @mkllnk.

I've re-staged the PR, re-enabled the toggle, and shared the details with the CQCM Staging Proxy. I've made sure this is a valid address:

<img width="1276" height="711" alt="image" src="https://github.com/user-attachments/assets/f911f691-9f29-4b71-a742-b5ae967c9be6" />

So, it should appear somewhere on this area:

<img width="1276" height="711" alt="image" src="https://github.com/user-attachments/assets/edd42e44-de6f-47a9-8fe7-d12eb923c8f4" />

However, I remember reading (somewhere!) that the portal is now in prod, so we'd only see this after merging the PR? @RachL can you confirm? We should be able to see it though in the backoffice of the map, if I understand correctly.

RachL commented:
@filipefurtad0 it might just be the cache indeed. Let's double check tomorrow morning though :+1:

rioug reviewed (approved):
Looks good :+1:

mkllnk commented:
@filipefurtad0, I implemented the webhook now but I ran out of time to test it. It's staged. You can give it another go or just wait till next week. I can test it on Monday. But if it works, then it can be merged and I added production configs, too. With some good luck, everything _could_ work.

RachL commented:
I've tested with enterprise 203916 and I don't see it for now on the proxy admin. I will follow and see how long it takes. Maybe there is an issue with the webhook, but I'm not 100% sure.

I've noticed the connected apps menu does not appear on producer enterprises, which can be a problem for DiscoRegen and waterloo, we would need to fix this before merging however it's less a priority than getting the webhook to work : the webhook would allow me to record a demo from pressing the agree and share button to the frontend.
I guess I can cheat if we don't manage: I could rename enterprises on both end to make it look like it has worked :thinking:  I need to test this out

RachL commented:
note I'm currently disabling disco regen on staging UK to test recording a demo, I will put it back up when I'm done

mkllnk commented:
Great! I'm now getting a successful response from the webhook. And I added coordinates to the address.

**But,** the enterprises now disappeared from the list. I don't know what's happening because it all looks correct from my side. What does SiB see?

mkllnk commented:
@RachL, can you find out which fields are not displayed and why? From my end it all looks correct and I don't see any errors.

RachL commented:
@mkllnk I've tried to share UK's super hub tamar Valley and this time it worked well :muscle: 

The only field that  I'm not seeing are latitude and longitude. But I see we have not stored them on OFN side. So that's maybe normal.

I will add them manually using data in montreal to keep the map centered and continue testing.

dacook commented:
@mkllnk does this require Gaetan's review for the newest commits; does it require more review from me, or does it require more work first?

mkllnk commented:
> the DM is super slow. can we do something about it or is it all on Sib's side?

That's on SiB's side. Well, we could pre-load some of the code earlier, before you get to this page but that's probably wasted effort. Lots of people would then load the component even when they don't use it.

There could be some kind of loading indicator. Maybe SiB could do that?

We could also try to serve the code from our server in the hope that it's quicker but SiB's CDN _should_ be fast, too, right? Their framework is just quite big.

mkllnk commented:
> How could we verify whether the webhook is triggered? Would we need to see the server logs? Or should it appear as a request on the console?

You won't see it in the browser console, no. If the webhook raises an error, we raise that error when updating the platforms as well. So the 500 error you see in the network requests indicates that the webhook failed. Bugsnag tells me that there was a timeout. Was the SiB proxy offline for a bit?

mkllnk commented:
> Given we might need to add longitude and latitude manually, maybe we want to leave it as is for now

I just added a proper address to Tamar Valley, in Montreal, including coordinates. Let's see if that solves the problem.

mkllnk commented:
@RachL 

> social media do not have enterprise linked

I can't see how to link from social media to enterprise. The enterprises are linking to social media already. Can you clarify with SiB what they are expecting?

mkllnk commented:
> catalog items do not have an enterprise in "managed by" field

The Connector doesn't support that field yet:

* https://github.com/datafoodconsortium/data-model-uml/blob/next/SUPPORTED.md

I'll ping Maxime to add it.

mkllnk commented:
I think that we should merge this pull request before it gets any bigger. The outstanding tasks seem to be a good chunk of work on their own, not directly related to the CQCM integration.

Regarding the images, I created a new issue:

* #13578
